action.yml

```
@@ -2,52 +2,51 @@ name: Update Deployment Board
 description: Adds deployment Issues to an Automated Deployment Project Board for visual tracking of what has been deployed to each environment.
 
 inputs:
-  github-token:
-    description: 'A token with permissions to create and update issues'
+  workflow-actor:
+    description: The GitHub user who triggered the workflow
     required: true
-  github-login:
-    description: 'The login associated with the github-token.  Defaults to github-actions'
+  token:
+    description: A token with permissions to create and update issues
```

```suggestion
    description: A token with permissions to create and update deployment objects
```
Do we know exactly what permission that is?